addons/hr_homeworking_calendar/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -98,7 +98,10 @@ patch(AttendeeCalendarCommonRenderer.prototype, {
     },
     headerTemplateProps(date) {
         if (this.props.model.scale === "month") {
-            return super.headerTemplateProps(date);
+            return {
+                ...super.headerTemplateProps(date),
+                showLocation: !this.context?.showLocation,
+            }
```

Comment:
Hi @jeh-odoo , a comment as you retake this one :)

Well, seems quite confusing to have something equal to its opposite!
maybe use `hideHomeworkingLocation` instead in appointment? and here have `!this.context?.hideHomeworkingLocation` ?

I'm not really a fan of the current solution using the context, but I wonder if using a prop-ish solution would be possible as we have both modules inheriting calendar and no bridge between them. Probably more thought to give to this one :thinking: